llvm/test/CodeGen/AArch64/sme2-intrinsics-fmlas16.ll

```diff
@@ -0,0 +1,462 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --filter-out "// kill:.*$" --version 4
+; RUN: llc -verify-machineinstrs < %s | FileCheck %s
+
```

Comment:
I believe we can add these tests in test/CodeGen/AArch64/sme2-intrinsics-fmlas.ll